.yarnrc.yml

```diff
@@ -13,50 +13,152 @@ logFilters:
+  - 'multicodec (deprecation)' # via cids
+
+  # MetaMask owned repositories brought in by other MetaMask dependencies that
+  # can be resolved by updating the versions throughout the dependency tree
+  - 'eth-sig-util (deprecation)' # via @metamask/eth-ledger-bridge-keyring
+  - '@metamask/controller-utils (deprecation)' # via @metamask/phishing-controller
+  - 'safe-event-emitter (deprecation)' # via eth-block-tracker and others
 
-npmRegistryServer: 'https://registry.npmjs.org/'
+  # @metamask-institutional relies upon crypto which is deprecated
+  - 'crypto (deprecation)'
 
+  # @metamask/providers uses webextension-polyfill-ts which has been moved to
+  # @types/webextension-polyfill
+  - 'webextension-polyfill-ts (deprecation)'
+
+  # Imported in @trezor/blockchain-link@npm:2.1.8, but not actually depended on
+  # by MetaMask
+  - 'ripple-lib (deprecation)'
+
+  # Brought in by ethereumjs-utils, which is used in the extension and in many
+  # other dependencies. At the time of this exclusion, the extension has three
+  # old versions of ethereumjs-utils which should be upgraded to
+  # @ethereumjs/utils throughout our owned repositories. However even doing
+  # that may be insufficient due to dependencies we do not own still relying
+  # upon old versions of ethereumjs-utils.
+  - 'ethereum-cryptography (deprecation)'
+
+  # Currently in use for the network list drag and drop functionality.
+  # Maintenance has stopped and the project will be archived in 2025.
+  - 'react-beautiful-dnd (deprecation)'
+  # New package name format for new versions: @ethereumjs/wallet.
+  - 'ethereumjs-wallet (deprecation)'
+
+  # The new trezor version breaks the webpack build due to issues with ESM and CommonJS
+  # Leading to this error on start: `Uncaught ReferenceError: exports is not defined`
+  # We temporarily ignore the audit failure until we can safely upgrade to the new version without breaking the webpack build
+  # Check Trezor 9.5.X Changelog for more info: https://github.com/trezor/trezor-suite/blob/develop/packages/connect/CHANGELOG.md
+  - '@trezor/connect-web (deprecation)'
+
+  # We temporarily ignore the deprecation notice to unblock ci
+  # Issue: @solana/web3.js version 2.0 is now @solana/kit! Remove @solana/web3.js@2 from your dependencies and replace it with @solana/kit.
+  # As needed, upgrade all of your @solana-program/* dependencies to the latest versions that use Kit.
+  - '@solana/web3.js (deprecation)'
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs
     spec: 'https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js'
   - path: .yarn/plugins/@yarnpkg/plugin-engines.cjs
-    spec: 'https://raw.githubusercontent.com/devoto13/yarn-plugin-engines/main/bundles/%40yarnpkg/plugin-engines.js'
+    spec: 'https://raw.githubusercontent.com/devoto13/yarn-plugin-engines/main/bundles/%40yarnpkg/plugin-engines.js'
```

Comment:
new line was missing before I added it in the other PR, don't come for me howard!